extensions/wikia/PortableInfobox/PortableInfobox.i18n.php

```diff
@@ -17,7 +17,27 @@
 	'portable-infobox-xml-parse-error-tag-name-mismatch' => 'Opening and ending tag mismatch',
 	'portable-infobox-xml-parse-error-tag-not-finished' => 'Premature end of tag',
 	'portable-infobox-xml-parse-error-infobox-tag-attribute-unsupported' => 'Attribute "$1" is not supported in
-	<infobox> tag'
+	<infobox> tag',
+
+	// Infobox Builder UI
+	'portable-infobox-builder-publish-button' => 'Publish',
+	'portable-infobox-builder-title' => 'Infobox Builder',
+	'portable-infobox-builder-edit-element-options-placeholder' => 'options placeholder',
+
+	// Infobox builder edit params
+	'portable-infobox-builder-edit-summary' => 'InfoboxBuilder',
+
+	// Infobox builder preview elements text placeholders
+	'portable-infobox-builder-infobox-title-element-placeholder' => 'I\'m Infobox Title',
+	'portable-infobox-builder-infobox-data-label-element-placeholder' => 'I\'m Label',
+	'portable-infobox-builder-infobox-data-value-element-placeholder' => 'I\'m Value',
```

Comment:
The two lines above would sound better with an indefinite article 'a' before the field name